### DISCUSSION TITLE
Strapi-plugin-documentation missing in node_modules?

### DISCUSSION THREAD

**Discussion by altaygileren** [strapi/strapi#6203] [Legacy Topics]
So I'm looking to fix this issue I'm having currently with Strapi where a nested relation is not showing the user object but just returning the user id.. I believe I found the solution but I could not follow the instructions as the package that looks like is needed is missing in the node_modules folder... since this was a change back from 18.2 and now I'm on currently 20.3 I was wondering if this solution still solves this issue or not? 

this is the current structure which I'm trying to fix... so when I map the Posts I want to be able to pull up the comments and all of the users information associated with those comments..

Post {
  post: 'this is my post'
  comments: [
    comment {
      post_comment: 'this is my comment',
      user: '13790783095'
    }
  ]
  user: {
    username: 'myusername',
    email: 'email@g.com'
  }
}

An example would be

User1 Post- "Hello world"
Comments
User2 comment- "Hi right back to you"

Currently the way this is set up is I am only shown the comment text and the user ID.. so I wouldn't be able to see the user2 username, email etc..

The link is below.

https://github.com/strapi/strapi/pull/4768

**Comment by lauriejim** [comment]
Hello! Can you please provide step to reproduce your issue.
I'm not sure to understand...

And can you check `Syling with Markdown` your message to make them more readable please.
Thank you.
